OpenDreamRuntime/Procs/Native/DreamProcNativeRoot.cs

```diff
@@ -854,7 +856,7 @@ public static DreamValue NativeProc_oviewers(DreamObject instance, DreamObject u
         public static DreamList params2list(string queryString) {
             queryString = queryString.Replace(";", "&");
             NameValueCollection query = HttpUtility.ParseQueryString(queryString);
-            DreamList list = new DreamList();
+            DreamList list = new DreamList(CurrentRuntime);
```

Comment:
params2list is used outside of proc calls (`HandlePacketTopic`), so `CurrentRuntime` may be null.

Reply:
Good point. I'll change HandlePacketTopic to create a (synchronous) DreamThread instead of directly calling the native proc.

If you wanted to avoid the DreamThread (which isn't exactly expensive), the solution would be to just move the functionality of params2list out of the native proc.